.github/workflows/test-mitosheet-frontend.yml

```diff
@@ -56,18 +62,24 @@ jobs:
         retention-days: 14
 
   test-mitosheet-frontend-streamlit:
-    runs-on: ubuntu-20.04
     timeout-minutes: 60
     strategy:
       matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
         python-version: ['3.10']
-        run_count: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        project: ['chromium', 'firefox', 'webkit', 'Google Chrome', 'Microsoft Edge']
```

Comment:
Looks like we are only testing on Python 3.6, 3.8, 3.10, maybe a next step for testing should be testing on all versions of Python we purport to support. Python 3.11 included